sass_processor/processor.py

```diff
@@ -65,7 +65,7 @@ def __call__(self, path):
         if not self.processor_enabled:
             return css_filename
         sourcemap_filename = css_filename + '.map'
-        if find_file(css_filename) and self.is_latest(sourcemap_filename):
+        if find_file(css_filename) and self.is_latest(sourcemap_filename, filename):
```

Comment:
for better readability, I prefer a more verbose name for `filename`. The variable should explain, what kind of file this is.

Reply:
would sourcefile be an ok name?

Reply:
I have a better idea, I will rework it to pass a base folder to is_latest